pkg/cmd/pipeline/logs_test.go

```diff
@@ -16,15 +16,16 @@ package pipeline
 
 import (
 	"errors"
-	"fmt"
 	"testing"
 	"time"
 
 	"github.com/AlecAivazis/survey/v2/core"
 	"github.com/AlecAivazis/survey/v2/terminal"
 	goexpect "github.com/Netflix/go-expect"
 	"github.com/jonboulle/clockwork"
+	"github.com/tektoncd/cli/pkg/cli"
 	"github.com/tektoncd/cli/pkg/helper/options"
+	htest "github.com/tektoncd/cli/pkg/helper/test"
```

Comment:
I feel this package would be better at `github.com/tektoncd/cli/test/prompt` (for the prompt part). It's clearer what it does : it's under test, so it's a test helper, and it's `prompt`, so it's prompt-related functions… *and* it wouldn't clash with `github.com/tektoncd/cli/test` package :angel:

Reply:
Thanks!! addressed 👍